presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -303,6 +305,7 @@ private Row createColumnStatsRow(String columnName, Type type, ColumnStatistics
             rowValues.add(NULL_DOUBLE);
             rowValues.add(toStringLiteral(type, columnStatistics.getRange().map(DoubleRange::getMin)));
             rowValues.add(toStringLiteral(type, columnStatistics.getRange().map(DoubleRange::getMax)));
+            rowValues.add(columnStatistics.getHistogram().map(Objects::toString).<Expression>map(StringLiteral::new).orElse(NULL_VARCHAR));
```

Comment:
We might want to show the histogram boundaries like [postgres](https://www.postgresql.org/docs/current/view-pg-stats.html)

Reply:
Currently it is left up to the implementation's `toString` method to do this. With the `UniformHistogram`, this does show the boundaries. Other planned histogram implementation such as with T-Digests or KLL sketches don't have "boundaries" in the traditional sense of binned histograms because they choose to represent the data differently. I think it makes more sense to leave it to the `ConnectorHistogram` implementation. If there were a binned representation I would want to print the boundaries.

Reply:
Sg